houston/queries.go

```diff
@@ -209,9 +209,9 @@ var (
 	}`
 
 	UserGetAllRequest = `
-	query GetUsers($userUuid: Uuid, $username: String, $email: String) {
-		users(userUuid: $userUuid, username: $username, email: $email) {
-			uuid
+	query GetUsers($userId: Id, $username: String, $email: String) {
```

Comment:
@schnie unused and invalid query: 
{"errors":[{"message":"Cannot query field \"createdAt\" on type \"Email\".","locations":[{"line":8,"column":5}],"extensions":
can I remove it?

Reply:
@andriisoldatenko ah, yea doesn't look like we've re-implemented it in Houston2. So there's no code in the CLI that calls that query?

If not, yea we can remove it.

Reply:
```
➜  astro-cli git:(switch-wording-from-uuid-to-id) grep --exclude-dir=vendor -r UserGetAllRequest .
./houston/queries.go:	//UserGetAllRequest = `
```